habitat-lab/habitat/tasks/rearrange/actions/oracle_nav_action.py

```diff
@@ -62,18 +63,20 @@ def reset(self, *args, **kwargs):
     def _get_target_for_idx(self, nav_to_target_idx: int):
         if nav_to_target_idx not in self._targets:
             action = self._poss_actions[nav_to_target_idx]
-            state = self._sim.capture_state(True)
-            task = action.init_task(
-                self._task.pddl_problem.sim_info, should_reset=True
-            )
-            target_pos = task.nav_target_pos
-            self._sim.set_state(state, True)
-            obj_entity = action.get_arg_value("obj")
+            nav_to_obj = action.get_arg_value("obj")
             obj_pos = self._task.pddl_problem.sim_info.get_entity_pos(
-                obj_entity
+                nav_to_obj
+            )
+            start_pos, _, _ = get_robot_spawns(
+                np.array(obj_pos),
+                0.0,
+                self._config.SPAWN_MAX_DIST_TO_OBJ,
```

Comment:
`SPAWN_MAX_DIST_TO_OBJ` should be lowercase. Btw, I can't find where its is defined/set.